packages/core-modules/contracts/modules/mocks/ElectionStorageMock.sol

```diff
@@ -43,21 +43,22 @@ contract ElectionStorageMock is ElectionStorage {
         _electionStore().seatCount = _electionStore().nextSeatCount;
         _electionStore().epochDuration = _electionStore().nextEpochDuration;
         _electionStore().nominationPeriodPercent = _electionStore().nextNominationPeriodPercent;
+        _electionStore().electionDataIndex++;
     }
 
     function getVoterVoteCandidatesMock(address addr) public view returns (address[] memory) {
-        ElectionData storage electionData = _electionStore().electionData;
+        ElectionData storage electionData = _electionData();
         VoteData storage voteData = electionData.votes[electionData.votesIndexes[addr]];
         return voteData.candidates;
     }
 
     function getVoterVoteVotePowerMock(address addr) public view returns (uint) {
-        ElectionData storage electionData = _electionStore().electionData;
+        ElectionData storage electionData = _electionData();
         VoteData storage voteData = electionData.votes[electionData.votesIndexes[addr]];
         return voteData.votePower;
     }
 
-    function getEpochStart() public view returns (uint) {
-        return _electionStore().epochStart;
+    function _electionData() private view returns (ElectionData storage) {
```

Comment:
What do you think about naming this `_currentElectionData()`? Here and in the non-mock contract of course.

Reply:
Fixed